README.md

```diff
@@ -35,32 +35,57 @@ before submitting your pull requests.
 
 ## Building
 
-Selenium uses a custom build system, aptly named
-[crazyfun](https://github.com/SeleniumHQ/selenium/wiki/Crazy-Fun-Build)
-available on all fine platforms (Linux, Mac, Windows).  We are in the
-process of replacing crazyfun with
-[buck](https://buckbuild.com/), so don't be alarmed if you
-see directories carrying multiple build directive files.
-For reference, crazyfun's build files are named *build.desc*,
-while buck's are named simply *BUCK*.
-
-Before building, ensure that you have Chrome browser installed and the
+### Bazel
+
+Bazel was built by the fine folks at Google. Like our previous systems, Bazel manages dependency 
+downloads, generates the Selenium binaries, executes tests and does it all rather quickly. Unlike
+what we used before, Bazel enjoys wide community support and updates outside of the Selenium project. 
+This enables the volunteers on this project to concentrate on Selenium code rather than maintaining 
+a separate build system in parallel.
+
+More detailed instructions for getting Bazel running are below, but if you can successfully get
+the java and javascript folders to build without errors, you should be confident that you have the 
+correct binaries on your system.
+
+### Older Build Systems
+
+Selenium previously used a custom build system, aptly named
+[crazyfun](https://github.com/SeleniumHQ/selenium/wiki/Crazy-Fun-Build). We then 
+replaced crazyfun with [buck](https://buckbuild.com/). All 3 build systems are 
+still operational, so don't be alarmed if you see directories carrying multiple 
+build directive files.
+
+- crazyfun build files are called *build.desc*,
+- buck build files are called *BUCK*.
+- There might be rake files about, as well
+
+### Before Building
+
+Ensure that you have Chrome browser installed and the
 [`chromedriver` ](https://sites.google.com/a/chromium.org/chromedriver/downloads) that matches
 your Chrome version available on your `$PATH`. You may have to update this from time to time.
 
-To build Selenium, in the same directory as this file:
+### Common Build Targets
+
+To build the most commonly-used modules of Selenium from source, execute this command from the root
+project folder:
 
 ```sh
-./go build
+bazel build java/... javascript/...
```

Comment:
Hooo-boy. This will do a lot of work in the JS files. I'm fine that that, but maybe warn people?